types/passport.socketio/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "connect-mongo": "^3.1.1"
+        "connect-mongo": "HoldYourWaffle/connect-mongo#fix-dt-external-error"
```

Comment:
what's the reason behind this change?

Reply:
See [this comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46576#issuecomment-669489207). It's mostly just a temporary thing to make CI tests pass until I know how to properly solve it.

I'll take a look at your other comments tomorrow.

Reply:
The best solution is to remove this dependency from @types/passport.socketio. It’s only used in tests, which has the quite unfortunate side effect of unnecessarily adding connect-mongo to the actual npm dependencies of @types/passport.socketio. If you remove it, you will solve your problem and save the 411 people per week who download @types/passport.socketio from bloating their node_modules 🎉